user-guide/objects/index.njk

```diff
@@ -0,0 +1,80 @@
+<p>To move an artboard you have to select it first and then drag it from its name or from an area without layers. </p>
+<p><strong>Tip:</strong> If you hold <kbd>Shift/⇧</kbd> while dragging you can do it from any part of the artboard, regardless where the layers are.</p>
+<p><img src="/img/artboards-move.gif" alt="artboards" /></p>
+<h4>Grids and prototyping</h4>
+<p>Artboards have two main particularities over the rest of the objects: you can set grids on them and you can connect them to create interactions.</p>
+<p><img src="/img/artboards-grid-int.png" alt="artboards" /></p>
+
+
+<h3 id="rectangles-ellipses">Rectangles and ellipses</h3>
+<p>Rectangle and ellipses are two basic, “primitive” geometric shapes that are really useful to kickstart your designs.</p>
+<p>Shortcuts are <kbd>E</kbd> for ellipses and <kbd>R</kbd> for rectangles.</p>
+<p>To know details about how to edit and modify them go to <a href="/user-guide/04-layer-basics/">Layer basics</a>.</p>
+<p><img src="/img/rect-ellipses.gif" alt="shapes" /></p>
+
+
+<h3 id="texts">Texts</h3>
+<p>To insert a text you have to activate the text tool first clicking on the icon at the toolbar or pressing <kbd>T</kbd>. Then you have to ways to create a text layer:</p>
+<ol>
+  <li><strong>Click</strong> to create a textbox without any specific dimensions.</li>
+  <li><strong>Drag</strong> to create a textbox with a fixed size.</li>
+</ol>
+<p><img src="/img/text-creation.gif" alt="text" /></p>
+<h4>Edit and style text content</h4>
+<p>Press <kbd>Enter</kbd> with a text layer selected to start editing the text content. You can style parts of the text content as a rich text.</p>
+<p><img src="/img/text-edit.gif" alt="text" /></p>
+<h4>Text options</h4>
+<p><img src="/img/text-options.png" alt="text" /></p>
+<ol>
+  <li><strong>Font family.</strong> You can choose any font from <a href="https://fonts.google.com/" target=”_blank”>Google Fonts</a>.</li>
+  <li><strong>Font size.</strong></li>
+  <li><strong>Font type.</strong></li>
+  <li><strong>Line height</strong> (in pixels).</li>
+  <li><strong>Letter spacing</strong> (in pixels).</li>
+  <li><strong>Text case:</strong> none, uppercase, lowercase, titlecase.</li>
+  <li><strong>Horizontal alignment:</strong> left, center, right, justify.</li>
+  <li><strong>Vertical alignment:</strong> top, center, bottom.</li>
+  <li><strong>Sizing:</strong> auto height, auto width, fixed size.</li>
+  <li><strong>Text decoration:</strong> none, underline, strikethrough.</li>
+</ol>
+
+
+<h3 id="paths">Paths (bezier)</h3>
+<p>A path is composed by two or more nodes that will form a vector shape. To draw a new path you have to activate the text path by clicking on the icon at the toolbar or pressing <kbd>P</kbd>. Then you have two ways to create a text shape:</p>
+<ol>
+  <li><strong>Click</strong> to create a new corner node.</li>
+  <li><strong>Click and drag</strong> to create a curved node.</li>
+</ol>
```

Comment:
Perhaps it's interesting to explain how to finish a path, leaving it open or closed.